include/stdinc.h

```diff
@@ -134,6 +134,7 @@ using std::flush;
 using std::min; 
 using std::max; 
 using std::ios; 
+using std::abs;
```

Comment:
make sure any remaining calls to `abs` use `std::abs` which does not convert between types